trestle/core/commands/create.py

```diff
@@ -0,0 +1,86 @@
+# -*- mode:python; coding:utf-8 -*-
+
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Trestle Create Command."""
+
+from ilcli import Command
+
+
+class CatalogCmd(Command):
```

Comment:
As above. I think we are doing even for this command a lot of 'special' behaviour and hard coding of OSCAL models. If we create a dict (maybe in the trestle.oscal module) which is something like

```python
top_level_models = {
'catalog': catalog.Catalog,
'profile': profile.Profile
...
}
I think we may be able to compress this down signficantly. The objective really is to have less boiler plate than is required. 

This won't be generically applicable, however, I would think would be very useful.

Reply:
Created issue #64.